setup.py

```diff
@@ -1079,6 +1103,32 @@ def print_box(msg):
     print("-" * (size + 2))
 
 
+def rename_torch_packages(package_list):
+    """
+    Create a dictionary from a list of package names, renaming packages where
```

Comment:
Why rename is necessary ? Maybe you can describe hi-level workflow of this build

Reply:
It's no longer needed. We can just delete this.